From 51ca1598081781e94b29220e827123007ed79f32 Mon Sep 17 00:00:00 2001
From: Gerd Hoffmann <kraxel@redhat.com>
Date: Wed, 10 Aug 2011 11:30:04 -0300
Subject: [RHEL6 qemu-kvm PATCH 10/10] Add user_print handler to qxl_screendump monitor command.

RH-Author: Gerd Hoffmann <kraxel@redhat.com>
Message-id: <1312975804-6692-1-git-send-email-kraxel@redhat.com>
Patchwork-id: 31212
O-Subject: [RHEL-6.2 kvm PATCH] Add user_print handler to qxl_screendump monitor command.
Bugzilla: 705070
RH-Acked-by: Alon Levy <alevy@redhat.com>
RH-Acked-by: Markus Armbruster <armbru@redhat.com>
RH-Acked-by: Amit Shah <amit.shah@redhat.com>

Otherwise the qmp monitor ignores us.  Oops.

bugzilla: 705070 - QMP: screendump command does not allow specification
                   of monitor to capture

Signed-off-by: Gerd Hoffmann <kraxel@redhat.com>
---
 qemu-monitor.hx |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

Signed-off-by: Eduardo Habkost <ehabkost@redhat.com>
---
 qemu-monitor.hx |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/qemu-monitor.hx b/qemu-monitor.hx
index 1084839..f8fe72c 100644
--- a/qemu-monitor.hx
+++ b/qemu-monitor.hx
@@ -295,6 +295,7 @@ ETEXI
         .args_type  = "id:s,filename:F",
         .params     = "id filename",
         .help       = "save screen from qxl device 'id' into PPM image 'filename'",
+        .user_print = monitor_user_noop,
         .mhandler.cmd_new = rhel6_qxl_do_screen_dump,
     },
 
-- 
1.7.3.2

